app/src/main/java/com/geekskool/dhobi/Models/Student.java

```diff
@@ -0,0 +1,94 @@
+package com.geekskool.dhobi.Models;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+
+/**
+ * Created by manisharana on 16/2/17.
+ */
+public class Student implements Serializable {
+
+    private String name;
+    private boolean isPreviousStudent;
```

Comment:
For consistency with the domain lexicon (aka vocabulary, aka glossary, aka dictionary, aka "shared language"), prefer the terms "new student" and "old student".